mxtoolbox_dns/help.md

```diff
@@ -1,11 +1,17 @@
 # Description
 
-[mxtoolbox_dns](https://www.mxtoolbox.com) is a Interface with MxToolBox DNS lookup API
+Manage email server configurations and troubleshoot email delivery issues.
```

Comment:
This is just a list of items. Should be a long description in paragraph form.

Reply:
Original link is missing as well

Reply:
Done